satellite/satellitedb/storagenodeaccounting.go

```diff
@@ -195,6 +195,48 @@ func (db *StoragenodeAccounting) QueryPaymentInfo(ctx context.Context, start tim
 	return csv, nil
 }
 
+// QueryNodeDailySpaceUsage returns slice of NodeSpaceUsage for given period
+func (db *StoragenodeAccounting) QueryNodeDailySpaceUsage(ctx context.Context, nodeID storj.NodeID, start time.Time, end time.Time) (_ []accounting.NodeSpaceUsage, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	// as entries are stored on daily basis we don't need
+	// to extract DATE from start_time
```

Comment:
this is not necessarily true, for older entries these might get condensed into a single entry.

Reply:
So that means we have a limit on how far we can go on the timescale, right? When we start condensing results 1 month, 2 month?

Reply:
It makes more sense to store some stats on the `storagenode` itself, and be able to ask satellite for it's point of view

Reply:
the accounting rollup is defined from `start_time` to the next entry for a particular node.